tests/phpunit/tests/media.php

```diff
@@ -3585,8 +3585,10 @@ public function test_wp_get_loading_attr_default( $context ) {
 			// and in the main query, and do not increase the content media count.
 			$this->assertSame( 'lazy', wp_get_loading_attr_default( 'wp_get_attachment_image' ) );
 
-			// Return `false` if in the loop and in the main query and it is the first element.
-			$this->assertFalse( wp_get_loading_attr_default( $context ) );
+			// Return `false` if in the loop and in the main query for first three element.
+			for ( $i = 0; $i < 3; $i++ ) {
+				$this->assertFalse( wp_get_loading_attr_default( $context ) );
```

Comment:
The new coding standard is the unit tests have message if their are more than one assert in a unit test. See https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions

This means if you change and existing test, you have update / add messages.

Reply:
Also, I wonder if using a loop here would be best for test debugging? I would think it is easier if there were three separate assertions. That way, when something starts breaking, you don't have to wrestle with the loop when debugging.